docker/generic/start_proxy.py

```diff
@@ -1080,6 +1088,8 @@ def enforce_conflict_args(args):
     if args.non_gcp:
         if args.service_account_key is None:
             return "If --non_gcp is specified, --service_account_key has to be specified, or GOOGLE_APPLICATION_CREDENTIALS has to set in os.environ."
```

Comment:
Please add unit test start_proxy.py. It should be easy.

Reply:
Done